cmd/controller-manager/app/options/generic.go

```diff
@@ -30,29 +29,27 @@ import (
 
 // GenericControllerManagerConfigurationOptions holds the options which are generic.
 type GenericControllerManagerConfigurationOptions struct {
-	Port                    int32
-	Address                 string
-	MinResyncPeriod         metav1.Duration
-	ClientConnection        componentbaseconfig.ClientConnectionConfiguration
-	ControllerStartInterval metav1.Duration
-	LeaderElection          componentbaseconfig.LeaderElectionConfiguration
-	Debugging               *DebuggingOptions
-	Controllers             []string
+	*kubectrlmgrconfig.GenericControllerManagerConfiguration
+	DebuggingFlag *DebuggingOptions
```

Comment:
why not just `Debugging` ?

Reply:
The origin thoughts is that do not override `Debugging` in `GenericControllerManagerConfiguration`. But seems override is OK.
fixed now